src/rsa.rs

```diff
@@ -238,9 +238,19 @@ impl PrivKeyFactory for RSAPrivFactory {
             Ok(k) => k,
             Err(_) => return err_rv!(CKR_WRAPPED_KEY_INVALID),
         };
+        /* Skip leading zeroes as they are not significant in the BN */
+        let modulus = rsapkey.modulus.as_bytes();
+        let mut skip = 0;
+        for i in 0..modulus.len() {
+            if modulus[i] != 0 {
+                break;
+            }
+            skip += 1;
+        }
+        let modulus = &modulus[skip..];
```

Comment:
I think this should be code in DerEncBigUint, in a function named something like: as_nopad_bytes() or similar. We do not want to repeat this code in multiple places.

Reply:
Indeed, this was mostly PoC to get the test working for now. Will do something like that.

Reply:
Moved to the separate function. Done so far only for the modulus as it has obvious conseqences, but might be worth doing also for other parameters.

Reply:
Yeah, should probably be done for all of them I guess.

Reply:
OK. Will add.